cluster/addons/dns/README.md

```diff
@@ -59,6 +59,21 @@ When enabled, pods are assigned a DNS A record in the form of `pod-ip-address.my
 
 For example, a pod with ip `1.2.3.4` in the namespace `default` with a dns name of `cluster.local` would have an entry: `1-2-3-4.default.pod.cluster.local`.
 
+
+####A Records and hostname Based on Pod Annotations - A Beta Feature in Kubernetes v1.3
+Currently when a pod is created it's hostname is the Pod's Name value.
+With v1.3, users can specify a Pod Annotation, pod.beta.kubernetes.io/hostname, to specify what the Pod's hostname should be.
```

Comment:
add an example:  e.g. `pod.beta.kubernetes.io/hostname: my-pod-name` will result in your Pod seeing `my-pod-name` as its hostname.